### PR TITLE
Change igsc dependency to match new igsc binary package name and version

### PR DESCRIPTION
**Pull request by mckees**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -26,7 +26,7 @@ Package: xpu-smi
 Architecture: amd64
 Depends:
  libze1 (>= 1.17.6),
- igsc (>=0.8.4),
+ intel-gsc (>=0.9.3),
```
Comment (hector-cao) [inline debian/control:29]: Does that mean that xpu-smi at the current version does not work with intel-gsc at version 0.8.4 ?

**Review by hector-cao — CHANGES_REQUESTED** [review]

File: debian/control (hunk 1 above)
Comment (mckees) [reply debian/control:29]: We will not have a version 0.8.4, so I'm just updating to the version that exists

Comment (mckees) [reply debian/control:29]: We will not have a version 0.8.4, so I'm just updating to the version that exists

**Review by hector-cao — APPROVED** [review]
LGTM

**Pull request merged**